packages/inter-protocol/src/auction/scheduler.js

```diff
@@ -204,6 +207,14 @@ export const makeScheduler = async (
     !liveSchedule || Fail`can't start an auction round while one is active`;
 
     assert(nextSchedule);
+    if (TimeMath.compareAbs(now, nextSchedule.startTime) > 0) {
+      console.warn(
+        `TIME skipped ahead. expected before ${q(
+          nextSchedule.startTime,
+        )}, but it is already ${now}`,
+      );
+      nextSchedule = computeRoundTiming(params, now);
+    }
```

Comment:
please comment here on why it's expected that time may jump forward and why this is the appropriate response. also make clear in the console output that this is not wholly unexpected and how the system is recovering.

e.g.
```suggestion
    // The clock tick may have arrived too late to trigger the next scheduled round.
    // For example, if the chain halted.  When this happens the oracle quote may
    // out of date and so must be ignored. Recover by returning deposits and
    // scheduling next round.
    if (TimeMath.compareAbs(now, nextSchedule.startTime) > 0) {
      console.warn(
        `Auction time jumped to ${now} before next scheduled ${q(
          nextSchedule.startTime,
        )}. Skipping that round.`,
      );
      nextSchedule = computeRoundTiming(params, now);
    }
```

Incidentally, why does `nextSchedule.startTime` need to be quoted? Why not `now`?

Reply:
Done.

`nextSchedule.startTime` is a timeRecord. None of its contents are sensitive, but it gets suppressed nonetheless.  `now` is a bigInt,  because #6003. I should `quote` it too, so it'll continuing being displayed after that issue has been addressed.